.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
     - npm install -g jslint
 
     # Install ESLint and plugins
-    - npm -g install eslint
+    - npm -g install eslint@3.8.1
```

Comment:
eslint 3.9 was released on the 28th of October which causes travis to fail. Version 3.8.1 was the last version that passed travis